entities/Session.js

```diff
@@ -0,0 +1,69 @@
+module.exports =  function(_, pg, db){
+	
+	function makeid(){
+        var text = "";
+        var possible = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
+
+        for( var i=0; i < 12; i++ )
+            text += possible.charAt(Math.floor(Math.random() * possible.length));
+        return text;
+    }
+
+    function createSession(data, callback){
```

Comment:
Please use any popular open source `session` middleware, a.e `express-session` with proper db-backend.
We don't want to maintain additional code that can be replaced with open source analogue.